apps/checkout/src/providers/AppConfigProvider/types.ts

```diff
@@ -0,0 +1,16 @@
+export type BrandingColors = Record<
+  | "buttonBgColorPrimary"
+  | "buttonBgColorHover"
+  | "borderColorPrimary"
+  | "errorColor"
+  | "successColor"
+  | "textColor"
+  | "buttonTextColor",
+  string
+>;
```

Comment:
Maybe not in this PR, so more like a sidenote - we can think of how to use the same common types for frontend, dashboard app and backend, so consistency will be assured.